config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml

```diff
@@ -44,6 +45,12 @@ periodics:
         value: "true"
       - name: KUBE_GCE_ENABLE_IP_ALIASES
         value: "true"
+      - name: NUM_WINDOWS_NODES
```

Comment:
When setting USE_RELEASE_NODE_BINARIES, what version of kubernetes will be used in this branch?